Yoast/YoastSEO.js
# Change feedback strings for most SEO assessments

## Summary

This PR can be summarized in the following changelog entry:

* Improves the feedback strings of SEO assessments.

## Relevant technical choices:

* Outbound links assessments: Makes sure the right feedback string is shown when there are no nofollowed links.

## Test instructions

This PR can be tested by following these steps:

* Trigger all SEO and readability assessments in all possible conditions as outlined here: https://github.com/Yoast/YoastSEO.js/wiki/Scoring-SEO-analysis

* Exceptions that don't need to be tested in this PR:
  * urlKeyword
  * keywordDistribution
  * keywordDensity/largestKeywordDistance
  * subheadingsKeyword
  * metadescriptionKeyword
* Make sure that all assessments have feedback strings according to the new format. This means:
  * In the beginning there is the assessment title with a link
  * In the end there is a call to action with a link (except when the bullet is green)

Fixes #1806
